docker/production/frontend/Dockerfile

```diff
@@ -3,8 +3,6 @@ WORKDIR /app
 COPY frontend/package.json frontend/yarn.lock ./
 RUN yarn install
 COPY ./frontend .
-# for some reason Eslint fails even though it passes outside the container
-RUN echo "**/*" > ./.eslintignore
```

Comment:
How did you fix it?

Reply:
I mean, do you know what was wrong?

Reply:
Yes, the problem was that eslint wasn't configured to validate `.vue` files, so even though I was running it multiple times manually and it passed, it didn't work when called via `yarn build` because it _somehow_ runs it properly.

So now, everytime you want to run eslint, it's enough to call `eslint --fix src` while inside `frontend`. You also need to check the logs from `web` to make sure that typescript compiles without any issues.